config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryconnectionconnections.bigqueryconnection.cnrm.cloud.google.com.yaml

```diff
@@ -308,6 +308,19 @@ spec:
                   current reported status reflects the most recent desired state of
                   the resource.
                 type: integer
+              observedState:
```

Comment:
All the comments in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1682/files#diff-508baa3dabf00328c18fcd828339759a13777f9c08481225ebffd438d071070f is still applicable to the CRD.

Reply:
Could you either turn `database`, `instanceId`, etc into reference fields accordingly, or maybe just ignore `cloudSql` and `cloudSpanner` fields completely in the service mapping?

Reply:
I will make them ignored fields for now. 
Support for cloudSql and cloudSpanner can be requested separately, https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/2149
But I don't want to hold up the current alpha release for that.